packages/cozy-client/src/models/countries.spec.js

```diff
@@ -0,0 +1,16 @@
+import { checkCountryCode } from './countries'
+
+describe('checkCountryCode', () => {
+  it('should return true for valid country codes', () => {
+    expect(checkCountryCode('FR')).toBe(true)
+    expect(checkCountryCode('US')).toBe(true)
+    expect(checkCountryCode('FRA')).toBe(true)
+    expect(checkCountryCode('USA')).toBe(true)
+  })
+
+  it('should return false for invalid country codes', () => {
+    expect(checkCountryCode('fr')).toBe(false)
```

Comment:
`fr` seems pretty legit to me,  I don't think this should be considered as invalid. We might force capitalization in `checkCountryCode` imo

Reply:
I made the modification, the function is no longer case sensitive 👍